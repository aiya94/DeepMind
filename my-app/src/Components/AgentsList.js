import React from "react";
import axios from "axios";
import "./SharingStyle.css";
import { Container, Row, Col, Badge } from "react-bootstrap";

export default class AgentsList extends React.Component {
  state = {
    agents: [],
    isLoading: true,
    errors: null
  };

  componentDidMount() {
    axios
      .get(`https://my-json-server.typicode.com/aiya94/DeepMind/agents`)
      .then(res => {
        const agents = res.data;
        this.setState({ agents, isLoading: false });
      })
      .catch(error => this.setState({ error, isLoading: false }));
  }

  render() {
    return (
      <div className="addMargin">
        <h2>
          AI <Badge variant="secondary">Page</Badge>
        </h2>
        <Container>
          <Row>
            {this.state.agents.map(agents => {
              return (
                <Col md={4} className="card">
                  <h4 className="headeling">{agents.name}</h4>
                  <p>{agents.description}</p>
                </Col>
              );
            })}
          </Row>
        </Container>
      </div>
    );
  }
}
