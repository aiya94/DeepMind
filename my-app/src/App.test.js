import React from "react";
import { render } from "@testing-library/react";

import AgentsList from "./Components/AgentsList";

it("renders AgentList Componenets", () => {
  const wrapper = render(<AgentsList />);
  expect(wrapper).toMatchSnapshot();
});
