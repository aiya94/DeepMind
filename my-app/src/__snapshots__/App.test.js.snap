// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders AgentComparison Componenets 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="addMargin"
      >
        <h2>
          AI 
          <span
            class="badge badge-secondary"
          >
            Comparison
          </span>
        </h2>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="addMargin"
    >
      <h2>
        AI 
        <span
          class="badge badge-secondary"
        >
          Comparison
        </span>
      </h2>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`renders AgentList Componenets 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="addMargin"
      >
        <h2>
          AI 
          <span
            class="badge badge-secondary"
          >
            Page
          </span>
        </h2>
        <div
          class="container"
        >
          <div
            class="row"
          />
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="addMargin"
    >
      <h2>
        AI 
        <span
          class="badge badge-secondary"
        >
          Page
        </span>
      </h2>
      <div
        class="container"
      >
        <div
          class="row"
        />
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`renders AgentsGeneral Componenets 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="addMargin"
      >
        <div
          class="container"
        >
          <div
            class="row"
          >
            <div
              class="col-sm-3"
            >
              <div
                class="flex-column nav nav-pills"
                role="tablist"
              >
                <div
                  class="nav-item"
                >
                  <a
                    aria-controls="left-tabs-example-tabpane-first"
                    aria-selected="true"
                    class="nav-link active"
                    data-rb-event-key="first"
                    href="#"
                    id="left-tabs-example-tab-first"
                    role="tab"
                  >
                    IMPALA
                  </a>
                </div>
                <div
                  class="nav-item"
                >
                  <a
                    aria-controls="left-tabs-example-tabpane-second"
                    aria-selected="false"
                    class="nav-link"
                    data-rb-event-key="second"
                    href="#"
                    id="left-tabs-example-tab-second"
                    role="tab"
                    tabindex="-1"
                  >
                    AlphaZero
                  </a>
                </div>
                <div
                  class="nav-item"
                >
                  <a
                    aria-controls="left-tabs-example-tabpane-third"
                    aria-selected="false"
                    class="nav-link"
                    data-rb-event-key="third"
                    href="#"
                    id="left-tabs-example-tab-third"
                    role="tab"
                    tabindex="-1"
                  >
                    R2D3
                  </a>
                </div>
              </div>
            </div>
            <div
              class="col-sm-9"
            >
              <div
                class="tab-content"
              >
                <div
                  aria-hidden="false"
                  aria-labelledby="left-tabs-example-tab-first"
                  class="fade tab-pane active show"
                  id="left-tabs-example-tabpane-first"
                  role="tabpanel"
                >
                  <div>
                    <canvas
                      height="150"
                      width="300"
                    />
                  </div>
                </div>
                <div
                  aria-hidden="true"
                  aria-labelledby="left-tabs-example-tab-second"
                  class="fade tab-pane"
                  id="left-tabs-example-tabpane-second"
                  role="tabpanel"
                >
                  <div>
                    <canvas
                      height="150"
                      width="300"
                    />
                  </div>
                </div>
                <div
                  aria-hidden="true"
                  aria-labelledby="left-tabs-example-tab-third"
                  class="fade tab-pane"
                  id="left-tabs-example-tabpane-third"
                  role="tabpanel"
                >
                  <div>
                    <canvas
                      height="150"
                      width="300"
                    />
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="addMargin"
    >
      <div
        class="container"
      >
        <div
          class="row"
        >
          <div
            class="col-sm-3"
          >
            <div
              class="flex-column nav nav-pills"
              role="tablist"
            >
              <div
                class="nav-item"
              >
                <a
                  aria-controls="left-tabs-example-tabpane-first"
                  aria-selected="true"
                  class="nav-link active"
                  data-rb-event-key="first"
                  href="#"
                  id="left-tabs-example-tab-first"
                  role="tab"
                >
                  IMPALA
                </a>
              </div>
              <div
                class="nav-item"
              >
                <a
                  aria-controls="left-tabs-example-tabpane-second"
                  aria-selected="false"
                  class="nav-link"
                  data-rb-event-key="second"
                  href="#"
                  id="left-tabs-example-tab-second"
                  role="tab"
                  tabindex="-1"
                >
                  AlphaZero
                </a>
              </div>
              <div
                class="nav-item"
              >
                <a
                  aria-controls="left-tabs-example-tabpane-third"
                  aria-selected="false"
                  class="nav-link"
                  data-rb-event-key="third"
                  href="#"
                  id="left-tabs-example-tab-third"
                  role="tab"
                  tabindex="-1"
                >
                  R2D3
                </a>
              </div>
            </div>
          </div>
          <div
            class="col-sm-9"
          >
            <div
              class="tab-content"
            >
              <div
                aria-hidden="false"
                aria-labelledby="left-tabs-example-tab-first"
                class="fade tab-pane active show"
                id="left-tabs-example-tabpane-first"
                role="tabpanel"
              >
                <div>
                  <canvas
                    height="150"
                    width="300"
                  />
                </div>
              </div>
              <div
                aria-hidden="true"
                aria-labelledby="left-tabs-example-tab-second"
                class="fade tab-pane"
                id="left-tabs-example-tabpane-second"
                role="tabpanel"
              >
                <div>
                  <canvas
                    height="150"
                    width="300"
                  />
                </div>
              </div>
              <div
                aria-hidden="true"
                aria-labelledby="left-tabs-example-tab-third"
                class="fade tab-pane"
                id="left-tabs-example-tabpane-third"
                role="tabpanel"
              >
                <div>
                  <canvas
                    height="150"
                    width="300"
                  />
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`renders App 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="App"
      >
        <div
          class="addMargin"
        >
          <h2>
            AI 
            <span
              class="badge badge-secondary"
            >
              Page
            </span>
          </h2>
          <div
            class="container"
          >
            <div
              class="row"
            />
          </div>
        </div>
        <div
          class="container"
        >
          <div
            class="row"
          >
            <div
              class="col-md-6"
            >
              <div>
                <canvas
                  height="150"
                  width="300"
                />
              </div>
            </div>
            <div
              class="col-md-6"
            >
              <div>
                <canvas
                  height="150"
                  width="300"
                />
              </div>
            </div>
          </div>
        </div>
        <div
          class="addMargin"
        >
          <h2>
            AI 
            <span
              class="badge badge-secondary"
            >
              Comparison
            </span>
          </h2>
        </div>
        <div
          class="addMargin"
        >
          <div
            class="container"
          >
            <div
              class="row"
            >
              <div
                class="col-sm-3"
              >
                <div
                  class="flex-column nav nav-pills"
                  role="tablist"
                >
                  <div
                    class="nav-item"
                  >
                    <a
                      aria-controls="left-tabs-example-tabpane-first"
                      aria-selected="true"
                      class="nav-link active"
                      data-rb-event-key="first"
                      href="#"
                      id="left-tabs-example-tab-first"
                      role="tab"
                    >
                      IMPALA
                    </a>
                  </div>
                  <div
                    class="nav-item"
                  >
                    <a
                      aria-controls="left-tabs-example-tabpane-second"
                      aria-selected="false"
                      class="nav-link"
                      data-rb-event-key="second"
                      href="#"
                      id="left-tabs-example-tab-second"
                      role="tab"
                      tabindex="-1"
                    >
                      AlphaZero
                    </a>
                  </div>
                  <div
                    class="nav-item"
                  >
                    <a
                      aria-controls="left-tabs-example-tabpane-third"
                      aria-selected="false"
                      class="nav-link"
                      data-rb-event-key="third"
                      href="#"
                      id="left-tabs-example-tab-third"
                      role="tab"
                      tabindex="-1"
                    >
                      R2D3
                    </a>
                  </div>
                </div>
              </div>
              <div
                class="col-sm-9"
              >
                <div
                  class="tab-content"
                >
                  <div
                    aria-hidden="false"
                    aria-labelledby="left-tabs-example-tab-first"
                    class="fade tab-pane active show"
                    id="left-tabs-example-tabpane-first"
                    role="tabpanel"
                  >
                    <div>
                      <canvas
                        height="150"
                        width="300"
                      />
                    </div>
                  </div>
                  <div
                    aria-hidden="true"
                    aria-labelledby="left-tabs-example-tab-second"
                    class="fade tab-pane"
                    id="left-tabs-example-tabpane-second"
                    role="tabpanel"
                  >
                    <div>
                      <canvas
                        height="150"
                        width="300"
                      />
                    </div>
                  </div>
                  <div
                    aria-hidden="true"
                    aria-labelledby="left-tabs-example-tab-third"
                    class="fade tab-pane"
                    id="left-tabs-example-tabpane-third"
                    role="tabpanel"
                  >
                    <div>
                      <canvas
                        height="150"
                        width="300"
                      />
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="App"
    >
      <div
        class="addMargin"
      >
        <h2>
          AI 
          <span
            class="badge badge-secondary"
          >
            Page
          </span>
        </h2>
        <div
          class="container"
        >
          <div
            class="row"
          />
        </div>
      </div>
      <div
        class="container"
      >
        <div
          class="row"
        >
          <div
            class="col-md-6"
          >
            <div>
              <canvas
                height="150"
                width="300"
              />
            </div>
          </div>
          <div
            class="col-md-6"
          >
            <div>
              <canvas
                height="150"
                width="300"
              />
            </div>
          </div>
        </div>
      </div>
      <div
        class="addMargin"
      >
        <h2>
          AI 
          <span
            class="badge badge-secondary"
          >
            Comparison
          </span>
        </h2>
      </div>
      <div
        class="addMargin"
      >
        <div
          class="container"
        >
          <div
            class="row"
          >
            <div
              class="col-sm-3"
            >
              <div
                class="flex-column nav nav-pills"
                role="tablist"
              >
                <div
                  class="nav-item"
                >
                  <a
                    aria-controls="left-tabs-example-tabpane-first"
                    aria-selected="true"
                    class="nav-link active"
                    data-rb-event-key="first"
                    href="#"
                    id="left-tabs-example-tab-first"
                    role="tab"
                  >
                    IMPALA
                  </a>
                </div>
                <div
                  class="nav-item"
                >
                  <a
                    aria-controls="left-tabs-example-tabpane-second"
                    aria-selected="false"
                    class="nav-link"
                    data-rb-event-key="second"
                    href="#"
                    id="left-tabs-example-tab-second"
                    role="tab"
                    tabindex="-1"
                  >
                    AlphaZero
                  </a>
                </div>
                <div
                  class="nav-item"
                >
                  <a
                    aria-controls="left-tabs-example-tabpane-third"
                    aria-selected="false"
                    class="nav-link"
                    data-rb-event-key="third"
                    href="#"
                    id="left-tabs-example-tab-third"
                    role="tab"
                    tabindex="-1"
                  >
                    R2D3
                  </a>
                </div>
              </div>
            </div>
            <div
              class="col-sm-9"
            >
              <div
                class="tab-content"
              >
                <div
                  aria-hidden="false"
                  aria-labelledby="left-tabs-example-tab-first"
                  class="fade tab-pane active show"
                  id="left-tabs-example-tabpane-first"
                  role="tabpanel"
                >
                  <div>
                    <canvas
                      height="150"
                      width="300"
                    />
                  </div>
                </div>
                <div
                  aria-hidden="true"
                  aria-labelledby="left-tabs-example-tab-second"
                  class="fade tab-pane"
                  id="left-tabs-example-tabpane-second"
                  role="tabpanel"
                >
                  <div>
                    <canvas
                      height="150"
                      width="300"
                    />
                  </div>
                </div>
                <div
                  aria-hidden="true"
                  aria-labelledby="left-tabs-example-tab-third"
                  class="fade tab-pane"
                  id="left-tabs-example-tabpane-third"
                  role="tabpanel"
                >
                  <div>
                    <canvas
                      height="150"
                      width="300"
                    />
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`renders BarChartComponent Componenets 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div>
        <canvas
          height="150"
          width="300"
        />
      </div>
    </div>
  </body>,
  "container": <div>
    <div>
      <canvas
        height="150"
        width="300"
      />
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`renders ImpalaChart Componenets 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div>
        <canvas
          height="150"
          width="300"
        />
      </div>
    </div>
  </body>,
  "container": <div>
    <div>
      <canvas
        height="150"
        width="300"
      />
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
